src/main/java/com/rcv/CastVoteRecord.java

```diff
@@ -116,6 +128,23 @@ void logRoundOutcome(
     Logger.log(Level.FINE, logStringBuilder.toString());
   }
 
+  Map<Integer, List<Pair<String, BigDecimal>>> getCdfSnapshotData() {
+    return cdfSnapshotData;
+  }
+
+  void logCdfSnapshotData(int round) {
```

Comment:
comments please

Reply:
I stopped adhering to the "comments for everything" rule a long time ago because it wasn't clear to me whether we'd actually end up having to follow that convention, it clutters up the code, and it's easy to fix later if we decide that we do need them. So, if it's OK with you, for now I'd prefer only to add them in places where you think it's actually unclear what's happening.

Reply:
Yeah, for sure.  I'm asking here and elsewhere because it's unclear what is happening.  Just a minimum of one line description of what is going on, and think of the audience as outside devs reading this for the first time.

Side note:  I'm continually surprised by how many veteran devs and managers don't encourage this practice.  A good dev will figure things out sooner or later, but the time savings from judicious comments is significant.

Reply:
Judicious, sure. Maybe I'm overreacting to the trauma of having to comment on every single variable and function.

How do you go on, when in your heart you begin to understand... there is no going back? There are some things that time cannot mend. Some hurts that go too deep.

Reply:
I am probably overreacting to the trauma of the code I'm working on at work :/